examples/using-wordpress/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "gatsby-example-using-wordpress",
+  "private": true,
+  "description": "Gatsby example site using the Wordpress source plugin",
+  "version": "1.0.0-beta.5",
+  "author": "Sebastien Fichot <fichot.sebastien@gmail.com>",
+  "dependencies": {
+    "babel-eslint": "^7.2.3",
+    "gatsby": "next",
+    "gatsby-link": "next",
+    "gatsby-plugin-google-analytics": "next",
+    "gatsby-plugin-manifest": "next",
+    "gatsby-plugin-offline": "next",
+    "gatsby-plugin-sharp": "next",
+    "gatsby-source-filesystem": "next",
+    "gatsby-transformer-sharp": "next",
+    "gatsby-source-wordpress": "next",
+    "graphql": "^0.10.3",
+    "hedron": "^0.7.1",
+    "highlight": "^0.2.4",
+    "json-stringify-safe": "^5.0.1",
+    "lodash": "^4.16.4",
+    "qs": "^6.4.0",
+    "react-helmet": "^5.1.3",
```

Comment:
install `gatsby-plugin-react-helmet` instead — that then handles the server rendering for you. Do the same with `react-plugin-styled-components`